### PR TITLE
Ban command

### PR DESCRIPTION
**Pull request by WillFJS**
Moved from #77

### CODE REVIEW

**Review by MidSpike — APPROVED** [review]
Looks good

**Review by rossloika — APPROVED** [review]
Looks a-okay to me :)

**Review by MidSpike — CHANGES_REQUESTED** [review]
Bug spotted, the owner of a server needs to handled

File: src/bot/commands/ban.js [hunk 1]
```diff
@@ -33,6 +33,11 @@ module.exports = {
             return;
         }
 
+        if (member.id === message.guild.ownerID) {
+            await message.reply('You aren\'t allowed to ban this user!');
```
Comment (MidSpike) [inline src/bot/commands/ban.js:37]:
```suggestion
            await message.reply('You aren\'t allowed to ban the owner of this server!');
```

**Review by MidSpike — CHANGES_REQUESTED** [review]

**Review by MidSpike — APPROVED** [review]

File: src/bot/commands/ban.js (hunk 1 above)
Comment (itsexdl) [reply src/bot/commands/ban.js]: w

**Pull request merged**